arangod/RocksDBEngine/RocksDBZkdIndex.h

```diff
@@ -44,8 +44,13 @@ class RocksDBZkdIndexBase : public RocksDBIndex {
   std::vector<std::vector<basics::AttributeName>> const& coveredFields()
       const override {
     // index does not cover the index attributes!
+    // return _coveredFields;
     return _storedValues;
```

Comment:
If the ZKD index has prefix fields, it will also cover them, provided it is chosen by the optimizer for eligible queries. So here we could return the union of stored values and prefix fields.

Reply:
As I wrote under ToDos
> Allow projections from the prefix fields. (Moved into separate tickets)

Reply:
Vale